master/buildbot/changes/gerritchangesource.py

```diff
@@ -298,17 +298,30 @@ def streamProcessStopped(self):
             self.startStreamProcess()
             self.streamProcessTimeout = self.STREAM_BACKOFF_MIN
 
+    def buildGerritCommand(self, *gerrit_args):
+        '''Get an ssh command list which invokes gerrit with the given args on the
+        remote host'''
+
+        cmd = [
+            "ssh",
+            "%s@%s" % (self.username, self.gerritserver),
+            "-p", str(self.gerritport)
+        ]
+
+        if self.identity_file is not None:
+            cmd += ["-i", self.identity_file]
```

Comment:
you probably want to use `cmd.extend(...)`

Reply:
Done, though they're equivalent.

Reply:
they are indeed, though the new code is consistent with what follows